actions/nodejs/secure-attestations-download/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "Secure attestations download for nodejs builder"
```

Comment:
I realize we only have the nodejs builder right now but I think most BYOB workflows could use this, no?

Also what situation were we expecting folks to use this? I guess if they need to do npm publish on their own?

Reply:
yes all folks can use it. I'm hesitant to create one for all ecosystems. If we ever need to change this, at least we'll be able to break only one of the builders, not all.

Reply:
Yeah, it's fine for now I guess.